app/src/forms/form/service.js

```diff
@@ -36,7 +43,12 @@ const service = {
       const obj = await service.readForm(formId);
       trx = await transaction.start(Form.knex());
 
-      await Form.query(trx).patchAndFetchById(formId, {name: data.name, description: data.description, public: data.public, active: data.active, labels: data.labels});
+      await Form.query(trx).patchAndFetchById(formId, {name: data.name, description: data.description, shortName: data.shortName, active: data.active, labels: data.labels});
```

Comment:
Style: some of these statements are long enough we should consider them being multi-line instead.

Reply:
Still in rough, will wait until API is settled and we start adding validation and security.  Then we will clean up the code.